static/app/components/onboarding/gettingStartedDoc/utils/useLoadGettingStarted.tsx

```diff
@@ -0,0 +1,57 @@
+import {useEffect, useState} from 'react';
+import * as Sentry from '@sentry/react';
+
+import type {Docs} from 'sentry/components/onboarding/gettingStartedDoc/types';
+import {
+  feedbackOnboardingPlatforms,
+  replayPlatforms,
+} from 'sentry/data/platformCategories';
+import type {PlatformKey} from 'sentry/types/project';
+
+export default function useLoadGettingStarted({
+  platformId,
+  platformPath,
+  replayPlatformCheck,
+  feedbackPlatformCheck,
```

Comment:
what if instead of having these two props, we did something like

`productType?: 'replay' | 'feedback'` and then do similar casing below